web/urls.py

```diff
@@ -20,6 +20,6 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('memberships/', include('memberships.urls')),
-    path('logout/', views.logout_view, name='logout')
+    # path('logout/', views.logout_view, name='logout')
```

Comment:
```suggestion

```

Reply:
Done!

Reply:
Sorry if it wasn't clear but by this I meant remove the commented out line